apps/comps/components/ongoing-competition.tsx

```diff
@@ -33,7 +33,7 @@ export const OngoingCompetition: React.FC<OngoingCompetitionProps> = ({
   return (
     <div className="bg-card p-8">
       <div className="mb-30 flex items-start justify-between">
-        <StringList strings={["ONGOING", ...competition.type]} />
+        <StringList strings={["ONGOING", competition.type]} />
```

Comment:
this was a bug that caused the `competition.type` to get split and look like `ONGOING / T / R / A / D / I / N / G`